lib/xenops_server.ml

```diff
@@ -2301,16 +2360,8 @@ module VM = struct
   let migrate context dbg id vdi_map vif_map vgpu_pci_map url =
     queue_operation dbg id (VM_migrate (id, vdi_map, vif_map, vgpu_pci_map, url))
 
-  let migrate_receive_memory _ dbg id memory_limit remote_instance_id c =
-    let is_localhost = instance_id = remote_instance_id in
-    let transferred_fd = Xcp_channel.file_descr_of_t c in
-    let op = VM_receive_memory(id, memory_limit, transferred_fd) in
-    (* If it's a localhost migration then we're already in the queue *)
-    if is_localhost then begin
-      immediate_operation dbg id op;
-      None
-    end else
-      Some (queue_operation dbg id op)
+  let migrate_receive_memory _ _ _ _ _ _ =
+    failwith "Unimplemented"
```

Comment:
Why do we still keep this function?

Reply:
Good question, but outside of the scope of this PR. Removing it entirely would involve coordination with xcp-idl I think.